src/config/searchWorker.ts

```diff
@@ -21,6 +21,43 @@ export default () => {
         DESC = 'DESC',
     }
 
+    // NOTE!: this is a copy from ResourceBrowser/Types
+    // imports don't get bundled with the service worker
+    enum NODE_SEARCH_KEYS {
```

Comment:
Suggestion: We can send stringified data in web-webworker please check if we can use that.
NOTE: Please fix the typo from service worker to web worker

Reply:
sending all the constants to search worker instead of redeclaring it